ext_mongodb.php

```diff
@@ -7,7 +7,7 @@
 
 interface Serializable extends Type
 {
-	function bsonSerialize() : array;
+	function bsonSerialize() : object;
```

Comment:
I have to do this, as the implementation through HNI needs to match the type, it is also required to match the type in the implementations. As you say that objects should be returned in the issue, I don't really have a choice.